projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -285,11 +330,21 @@ export class LfFieldTemplateContainerComponent extends LfFieldContainerDirective
     this.templateErrorMessage = undefined;
     if (open && !this.loadedTemplates) {
       try {
+        this.availableTemplates = [];
```

Comment:
why set availableTemplates to []?

Reply:
what would be the case where !this.loadedTemplates but this.availableTemplates has something in it.. Or is this to set it to empty instead of undefined? In that case, I think we should be checking for something else rather then setting it to a dummy empty value

Reply:
if getAvailableTemplatesAsync returns an error (but there is an preloaded template), this.availableTemplates has the preloaded template in it, but this.loadedTemplates is false.

Reply:
but it is weird that this.availableTemplates is not undefined if getAvailableTemplatesAsync throws